script/set-version

```diff
@@ -1,30 +1,30 @@
-#!/usr/bin/env bash
+#!/bin/sh -eux
 
 set -euo pipefail
 
-pvp="$1"
-semantic="$2"
-
-if [ -z "$pvp" ] || [ -z "$semantic" ]; then
+if [ $# -ne 2 ]; then
     echo "Usage: bump-version <VERSION> <DEPENDENCY>"
     exit 1
 fi
 
+pvp="$1"
+semantic="$2"
+
 specs=(
  amazonka/amazonka.cabal
  core/amazonka-core.cabal
  examples/amazonka-examples.cabal
  gen/amazonka-gen.cabal
  test/amazonka-test.cabal
- amazonka-s3-encryption/amazonka-s3-encryption.cabal
 )
 
 echo "Bumping version to $pvp and dependencies to == $semantic ..."
 
 for cabal in ${specs[@]}; do
-    sed -i"" "s/^\(version: *\).*$/\1$pvp/" $cabal
-    sed -i"" "s/^\(.*amazonka-[a-z0-9]* *== *\)\([0-9\.]\+\*\)$/\1$semantic.*/" $cabal
-    sed -i"" "s/^\(.*amazonka *== *\)\([0-9\.]\+\*\)$/\1$semantic.*/" $cabal
+    sed -e "s/^\(version: *\).*$/\1$pvp/;"  \
+        -e "s/^\(.*amazonka-[a-z0-9]* *== *\)\([0-9\.]\+\*\)$/\1$semantic.*/" \
+        -e "s/^\(.*amazonka *== *\)\([0-9\.]\+\*\)$/\1$semantic.*/" $cabal > $cabal.new
+    mv $cabal.new $cabal
```

Comment:
Echoing out to a new file then using mv gets around the incompatibilities with `sed` and inlace edits.

Reply:
:+1: